src/types_.py

```diff
@@ -252,12 +254,14 @@ class ContentChange(typing.NamedTuple):
     """Represents a change to the content.
 
     Attrs:
+        base: The content which is the base for comparison.
         old: The previous content.
         new: The new content.
     """
 
-    old: Content | None
-    new: Content | None
+    base: Content | None
+    old: Content
```

Comment:
**thought** I find the naming here a bit confusing. Why is server information the old one (I had to check in the code this to make sure that old was indeed what was coming from discourse)? Really, conceptually the oldest information is probably the base branch (since it does not have either the community contribution or the one proposed by the developer). discourse may even be the most updated version because of a community contribution. My suggestion would be to rename these fields as:

```
base => base
old => server
new => local
```

or

```
base => main
old => discourse
new => proposal
```

Reply:
Yeh I was debating this as I was writing the code myself. The reason these attributes are named the way they are is because it previously made sense and I tried to limit the changes as I was starting out. I'm inclined to change the attribute names now anyway since so far a lot of other similar changes have accumulated as well

Reply:
Should be fixed on the next push, I'll pick server and local to reduce the references to discourse